src/cesar/base_pass.rs

```diff
@@ -0,0 +1,45 @@
+use crate::cesar::{language::PropLang};
+use crate::cesar::config;
+use egg::*;
+
+pub static mut ASSUMPTIONS: String = String::new();
+
```

Comment:
Might be a good idea to add comments explaining the purpose of this class and how to use it.

Reply:
Done.